pkg/volume/gcepd/gce_pd.go

```diff
@@ -492,9 +492,13 @@ func (c *gcePersistentDiskProvisioner) Provision(selectedNode *v1.Node, allowedT
 	}
 
 	if fstype == "" {
-		fstype = "ext4"
+		if runtime.GOOS == "windows" {
```

Comment:
Hmm...so the default is set here (not in the apiserver)?

Reply:
No, this is where we first create PV object for dynamic provisioning. So at this time, we can check node os and decide the fsType based on this information

Reply:
Correct, fstype is not defaulted in the api because it's environment-specific.

Reply:
actually this change will not work. Provisioning is triggered from master controller, and checking the os here is actually the master os, not the node os. Discussed with @msau42 offline, there are two options here
1. set fsType when disk is "formatAndMount". 
2. set fsType when disk is attached to node. 
However both has problems because PV object is immutable after creation (for security reason, I think). To change this behavior requires some careful validation. Considering the cost, we agree that for Windows, we don't have default fsType. Windows users/admin must create a StorageClass which has the fsType specified. We can document this to make it clear.